### PR TITLE
Align with the latest W3C charter template

### LINKED ISSUE #55: Add charter history
The charter lacks a history section or changelog, e.g. highlighting the move from maintenance mode back into a more active mode.

### LINKED ISSUE #58: Success criteria ???
> The group will there ....

It looks like you rewrote the [boilerplate from the charter template](https://w3c.github.io/charter-drafts/charter-template.html).

Whenever that happens, I wonder why we're not seeing a PR back to the template - if the template needs to be improved, let's improve it. 

Meanwhile, the rewrite is ungrammatical.

### PR DESCRIPTION
**Pull request by iherman**
Changes are essentially editorial (update of links, formulation alignments, etc.)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-wg-charter/pull/49.html" title="Last updated on Jan 28, 2022, 2:34 PM UTC (9633bcb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-wg-charter/49/9a2a8d1...9633bcb.html" title="Last updated on Jan 28, 2022, 2:34 PM UTC (9633bcb)">Diff</a>

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -294,27 +302,28 @@ <h3>Timeline</h3>
...
-specification.
+        The Working Group will seek evidence of independent interoperable uses from at
+        least two independent implementations for each feature defined in the
+        specification.
       </li>
       <li>
-The group will there are sections in the specifications detailing any known
-security or privacy implications for implementers, Web authors, and end users.
+        The group will there are sections in the specifications detailing any known
+        security or privacy implications for implementers and end users.
```
Comment (Sakurann) [inline index.html]: I would rephrase the beginning of the sentence "The group will add sections in the specifications detailing any known security or privacy implications for implementers and end users." so that it becomes a sentence. Then I'd be happy to approve.

Comment (iherman) [reply index.html]: Oops, @Sakurann, you are right.

Actually... what I tried to do yesterday is to "combine" the previous section with the relevant section text in the template. From hindsight, that was a mistake, it just became more confusing. So I have just committed a version that takes over the boilerplate text from the template verbatim. Could you check that?

@brentzundel can you look at this again, too?

**Review by brentzundel — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html]: @Sakurann sorry, I did not make my wish explicit yesterday; can you give a thumbs up to the latest changes so that we can merge this?

File: index.html [hunk 2]
```diff
@@ -500,22 +500,21 @@ <h2>Licensing</h2>
...
         </p>
 
         <section id="history">
           <h3>
             Charter History
           </h3>
-          <p class="issue"><b>Note:</b>Display this table and update it when appropriate. Requirements for charter extension history are documented in the <a href="https://www.w3.org/Guide/process/charter.html#extension">Charter Guidebook (section 4)</a>.</p>
 
-          <p>The following table lists details of all changes from the initial charter, per the <a href="https://www.w3.org/Consortium/Process/#CharterReview">W3C Process Document (section 5.2.3)</a>:</p>
+          <p>The following table lists details of all changes from the initial charter, per the <a href="https://www.w3.org/Consortium/Process/#CharterReview">W3C Process Document (section 4.3, Advisory Committee Review of a Charter)</a>:</p>
```
Comment (samuelweiler) [inline index.html:517]: I prefer to see a history table that goes back to the beginning of the WG - something that shows the initial charter, the recharter into maintenance mode, and anything in between. The line for this recharter would name the new deliverables and summarize, very briefly, the change in scope.

Comment (iherman) [reply index.html:517]: (copying from https://github.com/w3c/vc-wg-charter/issues/55#issuecomment-1024239530)

Do you mean copy the whole of https://www.w3.org/2020/12/verifiable-credentials-wg-charter.html#about into this charter (with an additional line that is there)? I am fine with this, but I am a bit afraid it is a little bit over the top...

**Comment by iherman** [comment]
(Admin)
Per https://github.com/w3c/vc-wg-charter/pull/49#pullrequestreview-866148091 the discussion in #55 is now happening here; I have added #55 as a linked issue, ie, that will be closed if this PR is merged.

File: index.html (hunk 2 above)
Comment (iherman) [reply index.html:517]: @samuelweiler I have added the previous change table but, to reduce the size, I have removed the purely administrative changes from the previous table (change of staff contact, change of chairs). I think that should be o.k.

Comment (samuelweiler) [reply index.html:517]: You might ask comm for guidance/preferences.  I'd say that if you remove anything, put a note saying "more history can be found in the previous charter", and link it there (also).

Comment (iherman) [reply index.html:517]: Frankly, I do not want to loose my time on this. I have re-added those administrative history lines. The table is now complete.

**Review by brentzundel — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Sakurann) [reply index.html]: Approved! Thank you for fixing, Ivan)
